packages/suite/src/utils/suite/dom.ts

```diff
@@ -50,6 +50,9 @@ export const useKeyPress = (targetKey: string) => {
 
 export const copyToClipboard = (value: string, parent: HTMLDivElement | HTMLPreElement | null) => {
     try {
+        if (value.length <= 0) {
```

Comment:
remove this, disable button instead

Reply:
I believe that the user should not be allowed to copy an empty string, he may lose money due to this behavior

Reply:
anti-pattern in fintech, possible wrong bug, but i rewrote this